src/app/components/Embeds/OEmbed/index.tsx

```diff
@@ -1,10 +1,14 @@
 /* eslint-disable camelcase */
-import React, { useContext } from 'react';
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext } from 'react';
 import pathOr from 'ramda/src/pathOr';
-import { ServiceContext } from '../../../contexts/ServiceContext';
 import { RequestContext } from '../../../contexts/RequestContext';
+import { ServiceContext } from '../../../contexts/ServiceContext';
 import EmbedHtml from '../EmbedHtml';
 import EmbedError from '../EmbedError';
+import VjAmp from '../../../legacy/containers/Include/amp/VjAmp';
+import styles from '../EmbedHtml/index.styles';
```

Comment:
Is this ok to do (import styles from another component)? Or am I better to duplicate them?

Reply:
Is it worth creating a new folder and component called `AmpIframeEmbed` in the new `Embeds` folder. For now it would simply import the `VjAmp` component as you have here, but it would contain its own set of styles even if its duplicated.

Feels like we'll need to elevate the `VjAmp` component to the new component standards, so maybe that could be a good first step to doing it? The existing component isn't really specific to VJ in anyway, its just rendering and `amp-iframe` component so it could be used for any time we want to render an amp-iframe`.

Reply:
Thanks this makes sense. I've had a go at this.